templates/data_flow_job/_log.html.twig

```diff
@@ -0,0 +1,23 @@
+{% trans_default_domain 'data_flow' %}
```

Comment:
Should be `{% trans_default_domain 'data_flow_job' %}`.

Reply:
Done.